Cargo.toml

```diff
@@ -1,8 +1,12 @@
 [package]
 name = "vmm-sys-util"
 version = "0.1.0"
-authors = ["Jing Liu <jing2.liu@linux.intel.com>"]
-license = "Apache-2.0"
+authors = ["Intel Virtualization Team <vmm-maintainers@intel.com>"]
+description = "A system utility set for virtualization"
```

Comment:
This is not necessarily for virtualization. I think we should find a better description, but I can't think of any right now.

Reply:
Yep, it can be used by other scenarios either. What I am thinking is the name vmm-sys-util which describes the reason we create this crate?

Reply:
Maybe let's just make it as "A system utility set". The description is not web-view though.